spec/factories.rb

```diff
@@ -3,7 +3,7 @@
 FactoryGirl.define do
   factory :user do
     name "user"
-    email "user@test.com"
+    sequence(:email) { |n| "user{n}@test.com" }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.